packaging/linux/arch/DOT_SRCINFO.bin.in

```diff
@@ -12,7 +12,8 @@ pkgbase = keybase-bin
 	depends = gconf
 	depends = libxss
 	depends = gtk2
-	conflicts = keybase
+        depends = coreutils
+        conflicts = keybase
```

Comment:
coreutils is unnecessary, for the same reason that bash is unnecessary even though both scripts require it.

No sane system will fail to have either one installed, and on the off chance that some user had an **in**sane system, then they will have pre-existing problems due to missing `cat`, `chown`, `mkdir`, `rmdir` and `ln`.

Reply:
@eli-schwartz thanks, but things like `psmisc` are definitely required in other flavors of linux (for `killall`), so I thought it was better to be explicit.  Is there a downside to specifying it here?

Reply:
That depends.

I consider adding it to the PKGBUILD, to be a distraction from the real issue which is that you should document this in the comments for each executable script.
You can then choose whether to add those dependencies to all distro packaging, or only the distros that do not consider them to be base/essential packages. I *personally* would opt for the latter.

Also, adding coreutils as an extraneous, explicitly specified dependency could be seen as a lie, since you are essentially claiming that you are going out of your way to list all dependencies explicitly whether you need to or not... but, you still have unstated implicit dependencies on the following packages:

- bash
- psmisc
- sed
- shadow
- util-linux
- which

Reply:
These are all the commands currently used as far as I could tell in a quick sweep, and the Arch Linux method of finding which packages own them:

```
for i in env bash cat chown chmod mkdir rmdir ln killall which flock \
         sed nohup userdel useradd stat basename; do
    pacman -Qo /usr/bin/$i
done
```

Reply:
@eli-schwartz Thanks!  I think I'll try to keep this PR focused on the redirector stuff, and whatever new requirements are added by it, and we can look into cleaning up the dependencies in the future as we have time.

Reply:
My point is that no new requirements were added, only additional commands from pre-existing requirements, and the current state of the PKGBUILD makes absolutely no guarantees about implicit dependencies.

It therefore makes sense to **respect the current design** and only list dependencies the user isn't already expected to have. If listing implicit dependencies is something you want to do, don't start the job here and finish it at some undefined future date -- just ignore it for now, and raise the question of "is this wanted" at the aforementioned undefined future date.

> [...] adding coreutils as an extraneous, explicitly specified dependency could be seen as a lie, since you are essentially claiming that you are going out of your way to list all dependencies explicitly
This is what people will assume if they read the PKGBUILD without the context of this discussion, and see things like coreutils in the dependency list.

The packaging metadata for other flavors of Linux already lists psmisc as needed. Furthermore, adding coreutils to the Debian metadata is actually wrong and explicitly warned against in https://www.debian.org/doc/debian-policy/#dependencies

Whereas Arch Linux doesn't strictly warn against listing it anyways, but several Devs/Trusted Users are interested in making such a policy...